package.json

```diff
@@ -45,7 +46,7 @@
     "bundle": "browserify index.js -s ParseJapanese > parse-japanese.js",
     "postbundle": "esmangle parse-japanese.js > parse-japanese.min.js",
     "standard": "standard",
-    "test": "mocha --compilers js:espower-babel/guess test/**/*.js"
+    "test": "mocha --require babel-register 'test/**/*.js'"
```

Comment:
espower-babel is deprecated.
https://github.com/power-assert-js/espower-babel

`--compiers` flag is deprecated 
https://github.com/mochajs/mocha/wiki/compilers-deprecation